app/models/budget.rb

```diff
@@ -32,6 +30,14 @@ def period_end_date
     financial_year&.end_date
   end
 
+  def iati_type
+    IATI_TYPES[:original]
```

Comment:
If you use `IATI_TYPES.fetch(:original)` here, we'll get an error if that key doesn't exist. If the application expects certain pieces of data to exist it should raise an error as soon as it finds out the data is missing, rather than returning `nil` and causing a bug somewhere else.